static_src/components/main_container.jsx

```diff
@@ -9,17 +9,23 @@ import Disclaimer from './disclaimer.jsx';
 import Header from './header.jsx';
 import Login from './login.jsx';
 import LoginStore from '../stores/login_store.js';
+import OrgStore from '../stores/org_store.js';
+import SpaceStore from '../stores/space_store.js';
 import { Nav } from './navbar.jsx';
 
-function getState() {
-  return { isLoggedIn: LoginStore.isLoggedIn() };
+function stateSetter() {
+  return {
+    currentOrgGuid: OrgStore.currentOrgGuid,
```

Comment:
Definitely an improvement. It also helps to keep the stores as the single source of truth about the state of the data. If we're going to have the stores keep track of what app/space/org is the "current" one, then the components should use the store to get that information.